lib/open_api_2_json_schema.rb

```diff
@@ -60,6 +64,14 @@ def convert_schema(schema)
       end
     end
 
+    if schema.key?('allOf') && schema['allOf'].key?('$ref')
```

Comment:
Now that we don't merge schema anymore, no need to implement this field separately from the loop.